code/modules/clothing/gloves/color.dm

```diff
@@ -80,6 +80,19 @@
 	. = ..()
 	siemens_coefficient = pick(0,0,0,0.5,0.5,0.5,0.75)
 
+/obj/item/clothing/gloves/cut
+	desc = "These gloves would protect the wearer from electric shock.. if the fingers were covered."
+	name = "fingerless insulated gloves"
+	icon_state = "yellowcut"
+	inhand_icon_state = "ygloves"
+	siemens_coefficient = 1
+	permeability_coefficient = 1
+	resistance_flags = NONE
```

Comment:
the gloves should still be flammable. This probably shouldnt be overriden.

Reply:
aren't they made of rubber?